jetty-ee10/jetty-ee10-annotations/src/test/resources/web31false.xml

```diff
@@ -1,10 +1,10 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <web-app 
-   xmlns="http://xmlns.jcp.org/xml/ns/javaee" 
+   xmlns="https://jakarta.ee/xml/ns/jakartaee"
    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-   xsi:schemaLocation="http://xmlns.jcp.org/xml/ns/javaee http://xmlns.jcp.org/xml/ns/javaee/web-app_3_1.xsd"
+   xsi:schemaLocation="https://jakarta.ee/xml/ns/jakartaee https://jakarta.ee/xml/ns/jakartaee/web-app_6_0.xsd"
    metadata-complete="false"
-   version="3.1"> 
+   version="6.0">
```

Comment:
Why is a version=6.0 content inside a test file called web31?

Reply:
Note that if ee10 doesn't support 3.1 descriptors (@janbartel does it??) then we can probably delete the file and the test.   But 6.0 should be able to handle 5.0 descriptors at least, so we may need a new test/file